api/api.go

```diff
@@ -67,6 +67,13 @@ func (api *api) CreateApp(createAppRequest *CreateAppRequest) (*CreateAppRespons
 			"sub-wallets are currently not supported on your node backend. Try LDK or LND")
 	}
 
+	if slices.Contains(createAppRequest.Scopes, constants.SUPERUSER_SCOPE) {
+		if !api.cfg.CheckUnlockPassword(createAppRequest.UnlockPassword) {
+			return nil, fmt.Errorf(
+				"cannot create app with superuser permission without specifying unlock password")
```

Comment:
(minor) Can we have two checks? Because even if I give the wrong password it says I didn't specify

Reply:
hmm. We already have a `required` on the frontend so the user can't provide an empty password, and that's the only way right now. I'll update the error message to `"incorrect unlock password to create app with superuser permission"`

Reply:
Sorry I forgot to update the comment, previously there wasn't `required` so I ran into this, `incorrect unlock password` should be good 👍